tests/integration/upgrade-integration-test.sh

```diff
@@ -3,8 +3,9 @@ set -euo pipefail
 
 # This script is expected to be called from the root folder of Connaisseur
 
+RAND=$(head -c 5 /dev/urandom | hexdump -ve '1/1 "%.2x"')
```

Comment:
I'd add a comment here explaining why you're doing this

Reply:
done